base.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 module.exports = {
+  parser: 'babel-eslint',
```

Comment:
Again.

From [here](https://vuejs.github.io/eslint-plugin-vue/user-guide/#usage):

>##### How to use custom parser?
>If you want to use custom parsers such as [babel-eslint](https://www.npmjs.com/package/babel-eslint) or [typescript-eslint-parser](https://www.npmjs.com/package/typescript-eslint-parser), you have to use `parserOptions.parser` option instead of `parser` option. Because this plugin requires [vue-eslint-parser](https://www.npmjs.com/package/vue-eslint-parser) to parse `.vue` files, so this plugin doesn't work if you overwrote `parser` option.
```diff
- "parser": "babel-eslint",
  "parserOptions": {
+     "parser": "babel-eslint",
      "sourceType": "module"
  }
```